keen/explorer
# Upgrade to react 0.14

## What does this PR do? How does it affect users?

This PR upgrades the React version in explorer to 0.14. This means removal of a lot of deprecated methods, especially in tests. 

You can read through [this blog](http://reactkungfu.com/2015/10/upgrading-to-react-014/) to see some examples of the changes that needed to be made in order to upgrade the React version.

## How should this be tested

* Make sure there aren't any deprecated method warnings
* Make sure all tests pass
* ???
